ui/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "vscode-azureextensionui",
     "author": "Microsoft Corporation",
-    "version": "0.49.0",
+    "version": "0.50.0",
```

Comment:
There have been some cases in the past where this type of thing was a breaking change because the tests were implementing an interface or faking a method, It's unlikely for this scenario, though, so I don't really care.

Reply:
done